src/apps/bridge/app_config.cpp

```diff
@@ -1,6 +1,8 @@
 #include "app_config.h"
 #include "bridgeLog.h"
 #include "bridgePowerController.h"
+#include "rbrPressureProcessor.h"
```

Comment:
I see the other instances of this include are protected with the #define, should we do that here too?

Reply:
Yes! Good catch!